tests/nfv/prepare_env.pm

```diff
@@ -0,0 +1,61 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: openvswitch installation and CLI test
+#
+#   This test does the following
+#    - Installs needed packages (ovs, dpdk, git, ...)
+#    - Clones vsperf and dpdk repositories
+#    - Starts the systemd service unit
+#    - Executes a few basic openvswitch commands
```

Comment:
In vsperf_installation description we also had "Removes the lines to skip OVS, DPDK and QEMU compilation".
Otherwise LGTM

Reply:
ok to add it.